### DISCUSSION TITLE
Format converter for .raw and .dyr files

### DISCUSSION THREAD

**Discussion by thanever** [CURENT/andes#148] [Data Format] [answered]
I'd like to convert .raw and .dyr files into a single .xlsx file. It works to convert the .raw file into .xlsx, while running the command "andes run xxx.dyr --convert xlsx" throwed a error "Unable to determine case format." So how to realize this conversion?
Thanks.

**Comment by cuihantao** [comment, marked as answer]
The command to use is `andes run case.raw -a case.dyr -c`. The rationale is that the `dyr` file is an additional file to the `raw` file.

You can find more usage examples in https://github.com/cuihantao/andes/blob/master/examples/2.%20inspect_data.ipynb and https://docs.andes.app/en/stable/tutorial.html#pss-e-inputs.

**Comment by cuihantao** [comment]
Since this is not an issue, I'm converting it into a discussion. You can find the discussion thread from "Discussions" on the right of the "Pull Request" tab.
